src/app/commissioner_app.hpp

```diff
@@ -70,7 +70,7 @@ class CommissionerApp
     using Seconds      = std::chrono::seconds;
 
     static std::shared_ptr<CommissionerApp> Create(const std::string &aConfigFile);
-    ~CommissionerApp();
+    ~CommissionerApp() = default;
```

Comment:
There won't be any problem if `mCommissioner` is not stopped?

If that's true, you can just remove it.
```suggestion
```

Reply:
@Moandor-y  `mCommissioner` will try to stop itself in its destructor.

keep `~CommissionerApp() = default;` to make it explicit that we are not forgetting the destructor. It serves a comment.